Example/Backpack/View Controllers/TappableLinkLabelsViewController.swift

```diff
@@ -48,7 +48,7 @@ class TappableLinkLabelsViewController: UIViewController, BPKTappableLinkLabelDe
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        if self.style == .alternate {
+        if isAlternateStyleWithLightMode() {
```

Comment:
Now that we support dark mode, maybe when we enter this view we should just call `self.overrideUserInterfaceStyle = UIUserInterfaceStyleDark;`

wdyt?

Reply:
I thought that we were using that other background color for other reasons but if it was just to always try on dark scenarios then yeah we can definitely delete this code and always adopt dark mode for this particular screen. 
I'll do the change 👍

Reply:
Nice 👍

Reply:
This is no longer valid because our example app still needs to work on iOS prev to 13